zbsprimarystorage/zbsprimarystorage/zbsutils.py

```
@@ -38,9 +40,33 @@ def parse_cbd_path(path):
         snapshot = None
     return physical_pool, logical_pool, volume, snapshot
 
+"""
+ZBS Storage UUID Output Behavior:
+--------------------------------
+< v1.5.1        : UUID output NOT SUPPORTED
+v1.5.1 ~ v1.6.0 : UUID output enabled but returns status code 1 (ERROR)
+> v1.6.0        : Fixed to return status code 0 (SUCCESS)
+c.f. http://jira.zstack.io/browse/ZBS-327
+"""
+@in_bash
```

Ruff (0.12.2)

`in_bash` may be undefined, or defined from star imports

(F405)